public/app/features/dashlinks/module.ts

```diff
@@ -49,8 +49,10 @@ function dashLink($compile, $sanitize, linkSrv) {
         var linkInfo = linkSrv.getAnchorInfo(link);
         span.text(linkInfo.title);
         anchor.attr('href', linkInfo.href);
```

Comment:
I think it makes sense to add `href` only if dropdown is not used to prevent unnecessary link on button:
```js
span.text(linkInfo.title);
if (!link.asDropdown) {
  anchor.attr('href', linkInfo.href);
  sanitizeAnchor();
}
```

Reply:
Yeah sounds like a good idea. Will push a fix